.github/workflows/android-build.yml

```diff
@@ -15,11 +13,8 @@ jobs:
   build:
     if: github.repository == 'software-mansion/react-native-gesture-handler'
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        working-directory: [example, FabricExample]
```

Comment:
I'd assume that we also want to build the Expo app here, if so then the matrix would need to stay